### ISSUE TITLE
WebIO-related error message when using PlotlyJS backend

### ISSUE THREAD

**Issue by nhabedi** [JuliaPlots/Plots.jl#2572]
I'm not sure if this is the right place to report this issue because several libraries are involved, but I'll have to start somewhere... :)

If I open up a new Julia notebook and type
```
  using Plots
  plotlyjs()
```
I get a huge error message ("Error requiring WebIO from Plots") which can be seen below.  Nevertheless, the return value is `Plots.PlotlyJSBackend()` and I can then plot something, but the error message is annoying.

As WebIO is mentioned in the error message: If I start with a fresh notebook and type
```
  using WebIO
  node(:p, "Hello, Jupyter")
```
that works as expected and without error messages.

I'm on Windows, but I'm using Jupyter run from a Docker container based on https://hub.docker.com/r/jupyter/minimal-notebook/.  I get the same error in different browsers.  Details about the installed software and the error messages below.

* Ubuntu 18.04.4 LTS (Bionic Beaver)
* Python 3.7.6
* `jupyter --version` gives the following output:
    jupyter core     : 4.6.3
    jupyter-notebook : 6.0.3
    qtconsole        : not installed
    ipython          : 7.13.0
    ipykernel        : 5.1.4
    jupyter client   : 6.0.0
    jupyter lab      : 2.0.1
    nbconvert        : 5.6.1
    ipywidgets       : not installed
    nbformat         : 5.0.4
    traitlets        : 4.3.
* WebIO extension is enabled:
```
Known nbextensions:
  config dir: /home/jovyan/.jupyter/nbconfig
    notebook section
      webio-jupyter-notebook  enabled 
      - Validating: OK
```
* Julia is at version 1.4.0
* installed packages include:
  "Plots"                 => v"1.0.9"
  "PlotlyJS"              => v"0.13.1"
  "WebIO"                 => v"0.8.13"
  "IJulia"                => v"1.21.1"

And here is the stacktrace:

```
Warning: Error requiring WebIO from Plots:
LoadError: UndefVarError: WebIO not defined
Stacktrace:
 [1] top-level scope
 [2] eval at ./boot.jl:331 [inlined]
 [3] eval at /home/jovyan/.julia/packages/Plots/NVH6y/src/Plots.jl:1 [inlined]
 [4] (::Plots.var"#479#482")() at /home/jovyan/.julia/packages/Requires/qy6zC/src/require.jl:85
 [5] err(::Any, ::Module, ::String) at /home/jovyan/.julia/packages/Requires/qy6zC/src/require.jl:42
 [6] (::Plots.var"#478#481")() at /home/jovyan/.julia/packages/Requires/qy6zC/src/require.jl:84
 [7] withpath(::Any, ::String) at /home/jovyan/.julia/packages/Requires/qy6zC/src/require.jl:32
 [8] (::Plots.var"#477#480")() at /home/jovyan/.julia/packages/Requires/qy6zC/src/require.jl:83
 [9] listenpkg(::Any, ::Base.PkgId) at /home/jovyan/.julia/packages/Requires/qy6zC/src/require.jl:15
 [10] top-level scope at /home/jovyan/.julia/packages/Requires/qy6zC/src/require.jl:81
 [11] include(::Module, ::String) at ./Base.jl:377
 [12] include(::String) at /home/jovyan/.julia/packages/Plots/NVH6y/src/Plots.jl:1
 [13] top-level scope at /home/jovyan/.julia/packages/Plots/NVH6y/src/init.jl:61
 [14] eval at ./boot.jl:331 [inlined]
 [15] eval at /home/jovyan/.julia/packages/Plots/NVH6y/src/Plots.jl:1 [inlined]
 [16] (::Plots.var"#268#301")() at /home/jovyan/.julia/packages/Requires/qy6zC/src/require.jl:85
 [17] err(::Any, ::Module, ::String) at /home/jovyan/.julia/packages/Requires/qy6zC/src/require.jl:42
 [18] (::Plots.var"#267#300")() at /home/jovyan/.julia/packages/Requires/qy6zC/src/require.jl:84
 [19] withpath(::Any, ::String) at /home/jovyan/.julia/packages/Requires/qy6zC/src/require.jl:32
 [20] (::Plots.var"#266#299")() at /home/jovyan/.julia/packages/Requires/qy6zC/src/require.jl:83
 [21] #invokelatest#1 at ./essentials.jl:712 [inlined]
 [22] invokelatest at ./essentials.jl:711 [inlined]
 [23] foreach at ./abstractarray.jl:1919 [inlined]
 [24] loadpkg(::Base.PkgId) at /home/jovyan/.julia/packages/Requires/qy6zC/src/require.jl:22
 [25] #invokelatest#1 at ./essentials.jl:712 [inlined]
 [26] invokelatest at ./essentials.jl:711 [inlined]
 [27] require(::Base.PkgId) at ./loading.jl:930
 [28] require(::Module, ::Symbol) at ./loading.jl:922
 [29] top-level scope at /home/jovyan/.julia/packages/Plots/NVH6y/src/backends.jl:472
 [30] eval at ./boot.jl:331 [inlined]
 [31] _initialize_backend(::Plots.PlotlyJSBackend) at /home/jovyan/.julia/packages/Plots/NVH6y/src/backends.jl:471
 [32] backend at /home/jovyan/.julia/packages/Plots/NVH6y/src/backends.jl:176 [inlined]
 [33] plotlyjs(; kw::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at /home/jovyan/.julia/packages/Plots/H6y/src/backends.jl:33
 [34] plotlyjs() at /home/jovyan/.julia/packages/Plots/NVH6y/src/backends.jl:33
 [35] top-level scope at In[2]:1
 [36] eval at ./boot.jl:331 [inlined]
 [37] softscope_include_string(::Module, ::String, ::String) at /home/jovyan/.julia/packages/SoftGlobalScope/cSbw5/src/SoftGlobalope.jl:218
 [38] execute_request(::ZMQ.Socket, ::IJulia.Msg) at /home/jovyan/.julia/packages/IJulia/yLI42/src/execute_request.jl:67
 [39] #invokelatest#1 at ./essentials.jl:712 [inlined]
 [40] invokelatest at ./essentials.jl:711 [inlined]
 [41] eventloop(::ZMQ.Socket) at /home/jovyan/.julia/packages/IJulia/yLI42/src/eventloop.jl:8
 [42] (::IJulia.var"#15#18")() at ./task.jl:358
in expression starting at /home/jovyan/.julia/packages/Plots/NVH6y/src/backends/plotlyjs.jl:46
@ Requires /home/jovyan/.julia/packages/Requires/qy6zC/src/require.jl:44

```

**Comment by nhabedi** [comment]
I looked around a bit and it seems this was fixed in #2551.  Is that right?  So I'll probably have to wait for a new release...

**Comment by BoundaryValueProblems** [comment]
This problem has not been fixed completely. Please take a look at the latest posting in https://github.com/JuliaPlots/Plots.jl/issues/2462. Since this is a warning message, things seem to be working. Of course, it would be really great if this warning message is gone :-)
